src/config.ts

```diff
@@ -45,6 +45,19 @@ export const config = () => {
 
 export const KEY_CODES = { TAB_KEY: 9, ENTER_KEY: 13, ESCAPE_KEY: 27 };
 
+export const HTTP_CODES = {
```

Comment:
Can we put the `HTTP_CODES` in its own in the `types` directory. I think this would be useful for others as well.

Reply:
we should put KEY_CODES too I think.

Reply:
Moved to types/Common